build/common.sh

```diff
@@ -102,6 +103,7 @@ kube::build::get_docker_wrapped_binaries() {
           kube-scheduler,armel/busybox
           kube-aggregator,armel/busybox
           kube-proxy,gcr.io/google_containers/debian-iptables-arm:${debian_iptables_version}
+          gke-certificates-controller,busybox
```

Comment:
Good catch of bad copy/paste (I had only tested amd64)! Updated the other references.